certified-connectors/Map Pro/Readme.md

```diff
@@ -0,0 +1,41 @@
+## TeamsPro - Map Pro Connector
```

Comment:
Readme file must follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). Please review your readme file and add the required sections.